### DISCUSSION TITLE
Job table should be sorted by timestamp

### DISCUSSION THREAD

**Discussion by gaberger** [mikub/titanoboa#27] [Ideas]
Not sure what the implementation is but the job table should be sorted from most recent to oldest

**Comment by mikub** [comment]
@gaberger  Currently in "recent" tab I think the running jobs come on top regardless of the timestamp. I agree ti should be sorted,
The "archived" tab is sorted based on start time though.

Also making the table sortable based on various columns has been in my backlog for ages, maybe it's finally time to implement that  :sunglasses:
